Modules/System/Cues and KPIs/src/CuesAndKPIsImpl.Codeunit.al

```diff
@@ -259,4 +264,9 @@ codeunit 9702 "Cues And KPIs Impl."
               CueSetup.FieldCaption("Threshold 2"),
               CueSetup.FieldCaption("Threshold 1"));
     end;
+
+    [IntegrationEvent(false, false)]
```

Comment:
Having an integration event in the implementation would not work as expected.
Implementation codeunits, like this one, are internal so you won't be able to subscribe for this event.

To solve that, just move the event in the facade codeunit and call it here.

See [this example](https://github.com/microsoft/ALAppExtensions/blob/a499f0afcde09e0d836398a5d1abc2890b6ad7fb/Modules/System/Language/src/LanguageImpl.Codeunit.al#L19).

Reply:
Thank you for clarification. I have overlooked the internal property. I made necessary change. I hope that the change fits the prerequisites now.